ssl/t1_trce.c

```diff
@@ -728,7 +728,7 @@ static int ssl_print_extension(BIO *bio, int indent, int server, int extype,
         break;
 
     default:
-        BIO_dump_indent(bio, (char *)ext, extlen, indent + 2);
+        BIO_dump_indent(bio, (const char *)ext, extlen, indent + 2);
```

Comment:
is this  case necessary?  it does'nt just "upgrade" the parameter automatically?